lib/clientBuilder.js

```diff
@@ -41,6 +42,12 @@ function OktaAuthBuilder(args) {
     headers: args.headers
   };
 
+  if (!args.maxClockSkew && args.maxClockSkew !== 0) {
```

Comment:
Can you write a comment here about what this is used for? Also, what are we expecting as the value? Is it ms?

If ms, is 300 enough for the default value?